os/os.proto

```diff
@@ -94,6 +104,8 @@ service OS {
   //                              ...
```

Comment:
line 98:

Scenario 4 - On a dual Supervisor Target which requires one Install RPC per supervisor, and when one of the Supervisors already has the OS package but the other Supervisor is the target of the Install: